examples/sync/src/bin/server.rs

```diff
@@ -409,38 +519,53 @@ fn main() {
         info!(
             op_count = database.op_count(),
             root = %root_hex,
-            "Database ready"
+            "database ready"
         );
 
         // Create listener to accept connections
         let addr = SocketAddr::from((Ipv4Addr::LOCALHOST, config.port));
         let mut listener = match context.with_label("listener").bind(addr).await {
             Ok(listener) => listener,
             Err(e) => {
-                error!(addr = %addr, error = %e, "❌ Failed to bind");
+                error!(addr = %addr, error = %e, "❌ failed to bind");
                 return;
             }
         };
 
-        info!(addr = %addr, "Server listening");
+        info!(
+            addr = %addr,
+            operation_interval = ?config.operation_interval,
+            ops_per_interval = config.ops_per_interval,
+            "server listening and continuously adding operations"
+        );
+
+        let state = Arc::new(State::new(context.with_label("server"), database, config.seed));
```

Comment:
Is it worth making the example ~deterministic (did you feel this helped with debugging the bugs you found)?